torch/_dynamo/symbolic_convert.py

```diff
@@ -2278,6 +2304,7 @@ def __init__(
             code_options={k: getattr(code, k) for k in dir(code)},
             f_code=code,
             export=parent.export,
+            inline_depth=parent.inline_depth + 1,
```

Comment:
Can you make sure this works correctly w/ the higher order ops nested output_graph codepaths?